src/Billing/ErpExportsClient.php

```diff
@@ -0,0 +1,402 @@
+    /**
+     * @var string export filters vendors index
+     */
+    public const EXPORT_FILTERS_VENDORS = 'vendors';
+
+    /**
+     * @var string export filters programs index
+     */
+    public const EXPORT_FILTERS_PROGRAMS = 'programs';
+
+    /**
+     * @var string export filters marketplaces index
+     */
+    public const EXPORT_FILTERS_MARKETPLACES = 'marketplaces';
+
+    /**
+     * @var string export filters sequences index
+     */
+    public const EXPORT_FILTERS_SEQUENCES = 'sequences';
+
+    /**
+     * @var string export filters references index
+     */
+    public const EXPORT_FILTERS_REFRENCES = 'references';
+
+    /**
+     * @var string export filters resellerXspRefs index
+     */
+    public const EXPORT_FILTERS_RESELLER_XSP_REFS = 'resellerXspRefs';
+
+    /**
+     * @var string export filters resellerCompanyTags index
+     */
+    public const EXPORT_FILTERS_RESELLER_COMPANY_TAGS = 'resellerCompanyTags';
+
+    /**
+     * @var string export filters customerXspRefs index
+     */
+    public const EXPORT_FILTERS_CUSTOMER_XSP_REFS = 'customerXspRefs';
+
+     /**
+     * @var string export filters vendorSubscriptionIds index
+     */
+    public const EXPORT_FILTERS_VENDOR_SUBSCRIPTION_IDS = 'vendorSubscriptionIds';
+
+    /**
+     * @var string export filters friendlyNames index
+     */
+    public const EXPORT_FILTERS_FRIENDLY_NAMES = 'friendlyNames';
+
+    /**
+     * @var string export filters arrowSku index
+     */
+    public const EXPORT_FILTERS_ARROW_SKU = 'arrowSku';
+
+    /**
+     * @var string export filters date from index
+     */
+    public const EXPORT_FILTERS_DATE_FROM = 'from';
+
+    /**
+     * @var string export filters date to index
+     */
+    public const EXPORT_FILTERS_DATE_TO = 'to';
+
+    /**
+     * @return array|null
+     *
+     * @throws NotFoundException
+     * @throws PublicApiClientException|ReflectionException
+     */
+    public function getErpExportsColumns(): ?array
+    {
+        $response = $this->getErpExportsColumnsRaw();
+        $data = $this->decodeResponse($response);
+        $result = null;
+        if ($data['data'] && $data['data']['columns']) {
+            $result = $data['data']['columns'];
+        }
+
+        return $result;
+    }
+
+    /**
+     * @return string
+     *
+     * @throws NotFoundException
+     * @throws PublicApiClientException
+     */
+    public function getErpExportsColumnsRaw(): string
+    {
+        $this->path = "/erp/exports/columns";
+
+        return $this->get();
+    }
+
+    /**
+     * @return array|null
+     *
+     * @throws NotFoundException
+     * @throws PublicApiClientException|ReflectionException
+     */
+    public function getErpExportsTypes(): ?array
+    {
+        $response = $this->getErpExportsTypesRaw();
+        $data = $this->decodeResponse($response);
+        $result = null;
+        if ($data['data'] && $data['data']['exportTypes']) {
+            $result = $data['data']['exportTypes'];
```

Comment:
same here: you're using the response without reprocessing into an object so we won't have any autocompletion and won't know what fields are inside the response